### PR TITLE
add cv-Alex P

### PR DESCRIPTION
**Pull request by aperacaula**

### CODE REVIEW

File: staff/aperacaula/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <ul>
+        <li>Català (natiu)</li>
+        <li>Castellano (nativo)</li>
+        <li>Inglés (proficiency)</li>
+        <li>Português (avançado)</li>
+        <li>Français (basique)</li>
+    </ul>
+</body>
+
+</html>
```
Comment (MinaZhen) [inline staff/aperacaula/cv/index.html:47]: You forgot h4 and p tags!! I forgive you...

File: staff/aperacaula/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <br> tfn: 630075725</h3>
+    <h2>Experiencia Laboral</h2>
+    <ul>
+        <li>Feb 17-Aug 17: Intern in Strategy Planning Analysis in Principle Power Inc.</li>
+        <li>Feb 15-Aug 15: Scholar Researcher in National Fuel Cell Research Center</li>
+
+    </ul>
+    <h2>Educación</h2>
+    <ul>
+        <li>Double Master Degree Renewable Energy Engineering by Innoenergy
```
Comment (manuelbarzi) [inline staff/aperacaula/cv/index.html]: WARN cv is mainly in spanish, why this in english?

Comment (aperacaula) [reply staff/aperacaula/cv/index.html]: I chose to put it in the language of the Master's Degree, to give info about it, but perhaps you're right!

File: staff/aperacaula/cv/index.html (hunk 1 above)
Comment (aperacaula) [reply staff/aperacaula/cv/index.html:47]: haha, didn't know where to put them really, but you're totally right!

File: staff/aperacaula/cv/index.html (hunk 2 above)
Comment (aperacaula) [reply staff/aperacaula/cv/index.html]: DONE 👌🏻

File: staff/aperacaula/cv/index.html (hunk 1 above)
Comment (aperacaula) [reply staff/aperacaula/cv/index.html:47]: esto he optado por dejarlo así al final pero gracias 😉

**Review by MinaZhen — APPROVED** [review]

File: staff/aperacaula/cv/index.html (hunk 1 above)
Comment (MinaZhen) [reply staff/aperacaula/cv/index.html:47]: 👍

**Pull request merged**